stdlib/messaging/rabbitmq/src/main/java/org/ballerinalang/messaging/rabbitmq/util/ConnectionUtils.java

```diff
@@ -0,0 +1,212 @@
+                        RABBITMQ_CONNECTION_HEARTBEAT, LOGGER));
+            }
+            return connectionFactory.newConnection();
+        } catch (IOException | TimeoutException exception) {
+            LOGGER.error(CREATE_CONNECTION_ERROR, exception);
+            throw new BallerinaException(CREATE_CONNECTION_ERROR + " " + exception.getMessage(), exception);
+        }
+    }
+
+    /**
+     * Handles closing the given connection.
+     *
+     * @param connection   RabbitMQ Connection object.
+     * @param closeCode    The close code (See under "Reply Codes" in the AMQP specification).
+     * @param closeMessage A message indicating the reason for closing the connection.
+     * @param timeout      Timeout (in milliseconds) for completing all the close-related
+     *                     operations, use -1 for infinity.
+     * @param context      Context.
+     */
+    public static void handleClosingConnection(Connection connection, BValue closeCode, BValue closeMessage,
+                                               BValue timeout, Context context) {
+        if (RabbitMQUtils.checkIfNull(closeCode) && RabbitMQUtils.checkIfNull(closeMessage) &&
+                RabbitMQUtils.checkIfNull(timeout)) {
+            closeConnection(connection, context);
+        } else if (!RabbitMQUtils.checkIfNull(closeCode) && !RabbitMQUtils.checkIfNull(closeMessage) &&
+                !RabbitMQUtils.checkIfNull(timeout)) {
+            closeConnection(connection,
+                    Math.toIntExact(((BInteger) closeCode).intValue()),
+                    closeMessage.stringValue(),
+                    Math.toIntExact(((BInteger) timeout).intValue()), context);
+        } else if (!RabbitMQUtils.checkIfNull(closeCode) && !RabbitMQUtils.checkIfNull(closeMessage)
+                && RabbitMQUtils.checkIfNull(timeout)) {
+            closeConnection(connection,
+                    Math.toIntExact(((BInteger) closeCode).intValue()),
+                    closeMessage.stringValue(), context);
+        } else if (!RabbitMQUtils.checkIfNull(timeout)) {
+            closeConnection(connection,
+                    Math.toIntExact(((BInteger) timeout).intValue()), context);
+        } else if (!RabbitMQUtils.checkIfNull(closeMessage) && RabbitMQUtils.checkIfNull(closeCode)) {
+            throw new BallerinaException("If the closeMessage is given, closeCode cannot be null");
+        }
+    }
+
+    /**
+     * Closes the connection.
+     *
+     * @param connection RabbitMQ Connection object.
+     * @param context    Context.
+     */
+    private static void closeConnection(Connection connection, Context context) {
+        try {
+            connection.close();
+        } catch (IOException exception) {
+            LOGGER.error(CLOSE_CONNECTION_ERROR, exception);
+            RabbitMQUtils.returnError(CLOSE_CONNECTION_ERROR + " " + exception.getMessage(), context, exception);
```

Comment:
Instead of concat space everywhere shall we append that to variable value of `CLOSE_CONNECTION_ERROR`